plugins/core/rendering-info/index.js

```diff
@@ -144,7 +144,11 @@ function getPostRenderingInfoRoute(config) {
           request.params.target,
           requestToolRuntimeConfig,
           request.query.ignoreInactive,
-          itemStateInDb
+          itemStateInDb,
+          {
+            credentials: request.auth.credentials,
+            artifacts: request.auth.artifacts
+          }
```

Comment:
I guess that was the last function call that didn't support the credentials object, right?

Reply:
probably yes...